src/stores/RoomViewStore.tsx

```diff
@@ -443,6 +439,9 @@ export class RoomViewStore extends EventEmitter {
                 return;
             }
 
+            const viewRoomOpts: ViewRoomOpts = { buttons: [] };
+            ModuleRunner.instance.invoke(RoomViewLifecycle.ViewRoom, viewRoomOpts, this.getRoomId());
```

Comment:
I think a comment would help understand what's going on here. I was confused about why we still need `viewRoomOpts` since it looked like it was always set to `{ buttons: [] }`.
```suggestion
            const viewRoomOpts: ViewRoomOpts = { buttons: [] };
            // Allow modules to update the list of buttons for the room by updating `viewRoomOpts`.
            ModuleRunner.instance.invoke(RoomViewLifecycle.ViewRoom, viewRoomOpts, this.getRoomId());
```